README.md

```diff
@@ -128,11 +128,19 @@ app.register(SessionToken, Session);
 The `fusion-plugin-csrf-protection` module provides an api that matches the `fetch` api,
 and therefore can be registered on the standard `FetchToken` exported by `fusion-tokens`.
 
-* `fetch: (url: string, options: Object) => Promise` - A [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) implementation.
-* `expires: number` - Optional. Defaults to 86400 (seconds). When to expire the token.
-* `ignoredRoutes: Array<string>` - Optional. Defaults to `[]`. A list of paths that should not be gated by CSRF protection. For example `['/_errors']` would allow error logging requests to `/_errors` to be sent without a CSRF token.
-* `Session` - a Session plugin, such as the one provided by [`fusion-plugin-jwt`](https://github.com/fusionjs/fusion-plugin-jwt).
-  The Session instance should expose a `get: (key: string) => string` and `set: (key: string, value: string) => string` methods.
+#### Required dependencies
+
+Name | Type | Description
+-|-|-
+`FetchForCsrfToken` | `(url: string, options: Object) => Promise` | A [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) implementation.
+`SessionToken` | `Session` &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | A Session plugin, such as the one provided by [`fusion-plugin-jwt`](https://github.com/fusionjs/fusion-plugin-jwt).  The Session instance should expose a `get: (key: string) => string` and `set: (key: string, value: string) => string` methods.
```

Comment:
nit:  Do we need these spaces?

Reply:
Ugh, I'm torn with these.  They aren't necessary, but the formatting is pretty abysmal without them.  In most cases though, the tables work as I'd hope so this isn't common.

Reply:
Can you please double check? I tried removing them in Github editor and it seemed fine. But otherwise, I don't think it's an issue if we have to keep those

Reply:
Good call.  I had used a different renderer to test these out, which apparently has different opinions on the spacing.  Removed!